tests/conftest.py

```diff
@@ -2,6 +2,28 @@
 from bioimageio.spec import export_resource_package
 
 
+# set 'skip_<FRAMEWORK>' flags as global pytest variables,
+# to deselect tests that require frameworks not available in current env
+def pytest_configure():
```

Comment:
initially I thought "awesome", but now, I'm not sure if this is a good idea. First of all creating the development environment would ideally install all dependencies (imo) to run all tests. Then skipping some for missing deps is not necessary. The only issue might arise if there are conflicts that make it impossible to install everything into the same env.

Reply:
It also depends if we want tests to run for users who might only set up an environment with a subset of the dependencies.
I think it would also be quite useful to add another "blank" env for testing that does not contain any of the frameworks, to check that the module can be imported, and that base functionality works, with missing frameworks.